docs/src/content/pages/fields/blocks.mdoc

```diff
@@ -0,0 +1,31 @@
+---
+title: Blocks field
+---
+The `blocks` field is similar to the [array field](/docs/fields/array) in that you can create "Add one more" scenarios, but with the difference that it lets you define a separate field schema for each instance.
```

Comment:
Added this now since last review :) Please advise if you can think of improvements 🙏🏻